src/config/envs.js

```diff
@@ -2,7 +2,18 @@ import {ChainType} from '../enums';
 import {evaluate} from '../utils';
 
 export const env = process.env.NODE_ENV;
-export const appUrl = process.env.REACT_APP_URL;
+let appUrlPerEnv;
+switch (env) {
+  case 'development':
+    appUrlPerEnv = process.env.REACT_APP_URL_DEV;
+    break;
+  case 'test':
+    appUrlPerEnv = process.env.REACT_APP_URL_TEST;
+    break;
+  default:
+    appUrlPerEnv = process.env.REACT_APP_URL_PROD;
+}
```

Comment:
Use lookup table instead of the switch case.
```js
const appUrlMap = {
development: process.env.REACT_APP_URL_DEV,
test: process.env.REACT_APP_URL_TEST
}
export const appUrl = appUrlMap[env] || process.env.REACT_APP_URL_PROD;

```